app/controllers/home_controller.rb

```diff
@@ -1,2 +1,4 @@
 class HomeController < ApplicationController
+    def index
```

Comment:
Check identation here, Ruby standard is 1 tab (two spaces)

Reply:
I'll check this out!